docs/install.md

```diff
@@ -363,6 +363,10 @@ data:
 
 To customize the behavior of HA for the Tekton Pipelines controller, please refer to the related [documentation](developers/enabling-ha.md).
 
+## Overwriting the ThreadsPerController, QPS and Burst
```

Comment:
Suggest a slightly broader title of `Configuring Tekton Controller Performance`.

`ThreadsPerController`, `QPS`, and `Burst` are specific settings but we might one day add more ways to configure performance so placing them under a broader title makes sense I think.

Reply:
Done!